src/vaadin-grid.html

```diff
@@ -770,9 +777,19 @@
       _resizeHandler() {
         this._updateDetailsCellHeights();
         this._accessIronListAPI(super._resizeHandler, true);
+        this._updateScrollerMeasurements();
         this._updateHeaderFooterMetrics();
       }
 
+      _updateScrollerMeasurements() {
```

Comment:
Shouldn't this method be at `vaadin-grid-scroll-mixin`, as it is also using it there?